frontend/modules/model/model.sagas.ts

```diff
@@ -72,8 +72,15 @@ export function* downloadModel({ teamspace, modelId }) {
 	}
 }
 
-export function* onModelStatusChanged({ modelData: { status }, teamspace, project, modelId }) {
-	yield put(TeamspacesActions.setModelUploadStatus(teamspace, project, modelId, modelStatusesMap[status]));
+export function* onModelStatusChanged({ modelData, teamspace, project, modelId }) {
```

Comment:
I don't see this triggered, snackbar doesn't activate when the upload finishes...

Also, it would be nice to see the model name on the snackbar instead of just saying "model" as it can take a while before this happens.

Reply:
@dleszcz  can we do this?